package.json

```diff
@@ -229,18 +231,19 @@
     "prop-types": "15.7.2",
     "react-test-renderer": "17.0.2",
     "retry": "0.13.1",
-    "start-server-webpack-plugin": "2.2.5",
+    "start-server-nestjs-webpack-plugin": "2.2.5",
```

Comment:
It might be worth looking over the code of this fork just to double check it's secure, I would imagine there isn't that much to it?

Reply:
@andrewscfc think it'd be worth creating an issue to look at our dev-server setup?

Reply:
Yeah I think so, I suspect we can move away from the thing we are using now.